### ISSUE TITLE
Compatibility with Scene-Referred Workflow

### ISSUE THREAD

**Issue by natecostello** [bastibe/Fujifilm-Auto-Settings-for-Darktable#3]
I'm completely new to using darktable and RAW, so this may be a silly question:  The LUT module used by this script are display referred as far as I can tell.  Is this script compatible with a scene-referred workflow?

**Comment by bastibe** [comment]
Absolutely, yes! The LUTs are built to convert the filmic renders into film-sim-like renders.

However, in darktable the LUTs are applied after filmic, i.e. in the display-referred part of the pixelpipe. So in a sense it doesn't really matter if the pixelpipe before the LUT was scene-referred or display-referred as they both end up in more or less the same place.
